cea/optimization/individual_evaluation/individual_evaluation.py

```diff
@@ -24,9 +24,8 @@
 import cea.technologies.substation as sMain
 import cea.optimization.master.summarize_network as nM
```

Comment:
@shanshanhsieh in regards to the script, It should be located in a new folder describing what this tool does. It should not be inside optimization as it can be confused with modules that actually do some optimization work. May I suggest a new folder called simulation or workflow? what does the tool exactly do?

Reply:
Okay, i will move it to a folder called `supply` (in parallel to `demand`)
This tool simulates the performances of one supply system configuration (equivalent to one individual in the optimization).
The users can decide the cooling system configuration at the centralized and decentralized locations, as well as the buildings that are connected to the cooling network.